Scripts/DCS-BIOS/lib/modules/aircraft_modules/F-4E.lua

```diff
@@ -1452,6 +1452,37 @@ local AIRCRAFT_EFFECTS_DEVICE_ID = 63
 -- Target Designator System (AN-ASQ-153)
 local TARGET_DESIGNATOR_DEVICE_ID = 64
 
+-- Target Designator Set Control
+local TARGET_DESIGNATOR_SET_CONTROL = "Target Designator Set Control"
```

Comment:
I recommend separating this into `PLT Target Designator` and `WSO Target Designator`